tests/test_prescan_sub.py

```diff
@@ -0,0 +1,78 @@
+import glob
+import os
+
+import corgidrp.data as data
+from corgidrp.l1_to_l2a import prescan_biassub
+import corgidrp.mocks as mocks
+
+# Expected output image shapes
+shapes = {
+    'SCI' : {
+        True : (1200,2200),
+        False : (1024,1024)
+    },
+    'ENG' : {
+        True: (2200,2200),
+        False : (1024,1024)
+    }
+}
+
+def test_prescan_sub():
+    """
+    Generate mock input data and pass into prescan processing function
```

Comment:
It would be useful to make the docstring more informative. What does it test specifically? E.g., generate both ENG & SCI data, makes sure the output shapes are the size we expect, and cropping does not mess up the pointers in the Image class.